test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -147,8 +154,13 @@ func (hc *ContinuousHealthCheck) Start() {
 				defer cancel()
 				health, err := hc.esClient.GetClusterHealth(ctx)
 				if err != nil {
-					// TODO: Temporarily account only red clusters, see https://github.com/elastic/cloud-on-k8s/issues/614
-					// hc.AppendErr(err)
+					if IsMutationFromV6Cluster(hc.b) {
```

Comment:
I think technically the same thing can happen on a 7.x cluster (as you mentioned in the issue description) especially under load. It is just that the likelihood of such an event is much much lower because master elections are typically sub-second in 7.x. iiuc

I wonder if we will be able to observe this short unavailability in the tests.

Reply:
I don't know. Maybe worth waiting to see if that ever happens?